tethys_cli/__init__.py

```diff
@@ -61,6 +65,8 @@ def tethys_command_parser():
     add_quickstart_parser(subparsers)
     add_test_parser(subparsers)
     add_proxyapps_parser(subparsers)
+    if has_module("cookie_consent") or "sphinx" in " ".join(sys.argv):
+        add_cookie_parser(subparsers)
```

Comment:
Move the has_module check to the start of the add_cookie_parser() function instead of in here. Then this becomes just:


```suggestion
    add_cookie_parser(subparsers)
```